caproto/threading/client.py

```diff
@@ -1789,6 +1792,110 @@ def __del__(self):
             pass
 
 
+class Batch:
```

Comment:
This `Batch` syntax/functionality is even nicer than I had imagined - nice work!

Reply:
Good suggestion of syntax!